src/streaming/controllers/StreamController.js

```diff
@@ -521,6 +521,7 @@ function StreamController() {
         function onMediaSourceOpen() {
             // Manage situations in which a call to reset happens while MediaSource is being opened
             if (!mediaSource) return;
```

Comment:
Hi @jimmymic ,

could you, maybe, mutualize both lines in a single one with `||`? 

Thanks,

Reply:
Sure - updated